packages/next/src/utils.ts

```diff
@@ -1410,6 +1411,7 @@ export async function getPageLambdaGroups({
 }: {
   entryPath: string;
   config: Config;
+  functionsConfigManifest?: FunctionsConfigManifest;
```

Comment:
Lets make this required so its not missed on any calling function

```suggestion
  functionsConfigManifest: FunctionsConfigManifest;
```
We do this for `config` already because you can see some callers use `config = {}` if they don't have it.